### ISSUE TITLE
Increased Parameters on ./src/utils/openAI.ts

### ISSUE THREAD

**Issue by Big-jpg** [ddiu8081/chatgpt-demo#312]
### Clear and concise description of the problem

This is your current payload that you are sending to the OpenAI API:

```typescript  

 body: JSON.stringify({
    model,
    messages,
    temperature: 0.6,
    stream: true,
  }),

```

### Suggested solution

You could get more out of the API if you configure the additional parameters that the model supports:

such as this:

```typescript

  const payload: OpenAIStreamPayload = {
    model: 'gpt-3.5-turbo', // not much use changing this, since other models less than this are more costly and not as performant
    messages: messages, 
    temperature: process.env.AI_TEMP ? parseFloat(process.env.AI_TEMP) : 0.7, // using an env var here since it should be at 0.7
    max_tokens: 1024, // Hard upper limit to prevent people from DDoS-ing your endpoint
    top_p: 0.51, // see below
    frequency_penalty: 0.23, // see below
    presence_penalty: 0.23, // see below
    stream: true,
    user: body?.user, // pass in a cookie here, that way your own API account doesn't get banned due to your users actions
    n: 1, // see below
    logit_bias: {1925:5, 324:5}, //  see below
    }
  }

```

[For the Full API reference see here](https://platform.openai.com/docs/api-reference/chat/create)

For the other references above:

**top_p:** 
"An alternative to sampling with temperature, called nucleus sampling, where the model considers the results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered.

We generally recommend altering this or temperature but not both."

**frequency_penalty:**
Number between -2.0 and 2.0. Positive values penalize new tokens based on whether they appear in the text so far, increasing the model's likelihood to talk about new topics.

**presence_penalty:**
Number between -2.0 and 2.0. Positive values penalize new tokens based on their existing frequency in the text so far, decreasing the model's likelihood to repeat the same line verbatim.

**n**
How many chat completion choices to generate for each input message.
(putting this value higher than 1 is hilarious but not really useful.

// logit_bias
Modify the likelihood of specified tokens appearing in the completion.

Accepts a json object that maps tokens (specified by their token ID in the tokenizer) to an associated bias value from -100 to 100. Mathematically, the bias is added to the logits generated by the model prior to sampling. The exact effect will vary per model, but values between -1 and 1 should decrease or increase likelihood of selection; values like -100 or 100 should result in a ban or exclusive selection of the relevant token.

You need to define this as a Map Object and pass Key Value Pairs to it.
[Use this site to determine your tokens](https://platform.openai.com/tokenizer?view=bpe)

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ddiu8081/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/ddiu8081/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by yzh990918** [comment]
The v2 version will support customization
